.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "8"
+install:
+  - npm install
+  - npm run test
```

Comment:
I don't think we can these, they require a browser.

Reply:
Removed.

Reply:
Headless Chrome @SimonSchick ?
https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-the-Chrome-addon-in-the-headless-mode

Reply:
I made the headless chrome browser work. Here are the references to make it work: https://developers.google.com/web/updates/2017/06/headless-karma-mocha-chai
https://github.com/travis-ci/travis-ci/issues/8836